src/stores/MessageStore.js

```diff
@@ -9,17 +9,6 @@ let CHANGE_EVENT = 'change';
 
 let _messages = {};
 
-function _addMessages(rawMessages) {
```

Comment:
Thanks for the review, I rechecked, this method was not used anywhere in the code and doesn't affect any other component. Everything works fine when this function is removed. 
Please check by pulling my PR.

Thanks